src/app/CommandHandler.h

```diff
@@ -559,13 +552,17 @@ class CommandHandler : public Messaging::ExchangeDelegate
     CommandPathRegistry * mCommandPathRegistry = &mBasicCommandPathRegistry;
     Optional<uint16_t> mRefForResponse;
 
+    chip::Callback::Callback<OnResponseSenderDone> mResponseSenderDone;
```

Comment:
```suggestion
    Callback::Callback<OnResponseSenderDone> mResponseSenderDone;
```

Reply:
Because `Callback` is a inner class of `CommandHandler` we do need to be more explicit here